roles/common/templates/networkpolicy.yml.j2

```diff
@@ -0,0 +1,14 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: "{{ meta.name }}-networkpolicy-{{ trunc_uuid }}"
+  namespace: '{{ operator_namespace }}'
+spec:
+  podSelector: 
+    matchLabels:
+      type: uperf-bench-server-{{ trunc_uuid }}
```

Comment:
The reason I mentioned my comment is because of the "uperf"- here.

Reply:
done